.github/workflows/build-artifact.yaml

```
@@ -0,0 +1,66 @@
+
+          mkdir -p dist
+          tar czf "dist/flightctl-${{ github.event.pull_request.number }}.tar.gz" \
+            --exclude='dist' \
+            --sort=name \
+            --mtime="@0" \
+            --owner=0 \
+            --group=0 \
+            --no-same-permissions \
+            --no-same-owner \
+            --numeric-owner
+          
```

yamllint (1.35.1)

[error] trailing spaces

(trailing-spaces)